tianshou/utils/log_tools.py

```diff
@@ -0,0 +1,27 @@
+from torch.utils import tensorboard
+import threading
+
+
+class SummaryWriter(tensorboard.SummaryWriter):
+    """
+    A more convenient Summary Writer, you can get the same instance of summary
+    writer every where after you have create one.
+
+    """
+    _mutex_lock = threading.Lock()
+    _default_key = None
+
+    @classmethod
+    def get_instance(cls, key: str = 'default', *args, **kwargs):
```

Comment:
Maybe you can specify the ``args, kwargs`` (like ``log_dir``). They make coding much easier.